float64 x_center 
float64 y_center

float64 r

float64 a
float64 b
float64 angle

float64 height_factor
float64 object_height

int16 cls
float64[] probabilities

float64 Final_Likelihood

